policyengine_us/parameters/gov/states/ri/tax/income/adjusted_gross_income/subtractions/child_tax_rebates/child_tax_rebates_amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: RI Child Tax Rebates Amount for Each Child
+metadata:
+  label: RI child tax rebates amount
+  period: year
+  unit: currency-USD
+  reference:
+    - title: R.I. Gen. Laws § 44-30-103  44-30-103. Child tax rebates.
+      href: http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-30/44-VI/44-30-103.htm
+
```

Comment:
need to add 2021 and 2022 tax forms -> should be 3 separate references

Reply:
all yaml

Reply:
The child rebates does not have 2021 information